docs/serving/autoscaling/kpa-specific.md

```diff
@@ -264,3 +264,59 @@ This setting determines the maximum ratio of existing to desired pods. For examp
         autoscaler:
           max-scale-down-rate: "4.0"
     ```
+
+### Scale Down Delay
```

Comment:
I'm not sure we should hide this feature inside the KPA page since it's a pretty useful feature. It seems more confusing than helpful for a reader to have to guess to look in the KPA page rather than seeing it with the other scale related properties. The other things on the KPA page are very specific to KPA, rather than just not implemented in HPA yet -- I'd prefer we just mark that this does not currently work with HPA.

Reply:
:+1: I will move this back.